src/Datadog.Trace.ClrProfiler.Managed/Integrations/ElasticsearchNet5Integration.cs

```diff
@@ -57,6 +56,7 @@ public static class ElasticsearchNet5Integration
                        .Start(moduleVersionPtr, mdToken, opCode, methodName)
                        .WithConcreteType(pipelineType)
                        .WithMethodGenerics(genericArgument)
+                       .WithNamespaceAndNameFilters("Elasticsearch.Net.ElasticsearchResponse`1", "Elasticsearch.Net.RequestData")
```

Comment:
I guess here the NamespaceAndNameFilters doesn't match when we look at Generic types. Would there be a small amount of work that could make these match, so then we could maybe only specify the list once, and therefore improve consistency?

Reply:
I'm fine with how these look already but I think it would be a positive improvement to only define them once and match in both scenarios, if it's feasible to do.

Reply:
I agree, I would like to have a better way to make this DRY